src/model.hpp

```diff
@@ -38,9 +38,11 @@ class Model {
     mutable std::shared_mutex modelVersionsMtx;
 
     /**
-         * @brief Update default version
-         */
-    void updateDefaultVersion();
+      * @brief Update default version
+      *
+      * @param ignoredVersion Version that should not be taken into account
```

Comment:
I would be good to improve this description. Why a version should be ignored?

Reply:
How would you like to improve the description?

Reply:
maybe like this: to exclude them from being selected as the default version (for example when the version is to be retired)

Reply:
changed